changelogs/fragments/1247-volser-parsing-leading-dash-bugfix

```diff
@@ -0,0 +1,5 @@
+bugfix:
```

Comment:
Heads up Ketan, I ran  `antsibull-changelog lint` and it looked fine. Turns out that this file does not have a .yml extension. Once I added it I was able to lint it : `changelogs/fragments/1247-volser-parsing-leading-dash-bugfix.yml:0:0: invalid section: bugfix`

Section should be named bugfixes not bugfix